image/tarexport/save.go

```diff
@@ -154,6 +155,15 @@ func (s *saveSession) save(outStream io.Writer) error {
 			RepoTags: repoTags,
 			Layers:   layers,
 		})
+
+		parentID, _ := s.is.GetParent(id)
+		parentLinks = append(parentLinks, parentLink{id, parentID})
```

Comment:
Curious why golint doesn't complain about no field names being specified in the initializer.